basic_module/account/src/types.rs

```diff
@@ -0,0 +1,95 @@
+use ccrypto::blake256;
+use ckey::{Ed25519Public as Public, NetworkId, Signature};
+use primitives::H256;
+
+#[allow(dead_code)]
+pub type ErroneousTransactions = Vec<SignedTransaction>;
+
+#[derive(Clone)]
+pub struct Account {
+    pub balance: u64,
+    pub sequence: u64,
+}
+
+// Temporary measures for build.
+// This function will be removed when the structure of the context and conversion logic is completed.
+impl From<Vec<u8>> for Account {
+    fn from(_vec: Vec<u8>) -> Account {
+        Account {
+            sequence: 0,
+            balance: 0,
+        }
+    }
+}
+
+impl Default for Account {
+    fn default() -> Self {
+        Account {
+            balance: 0,
+            sequence: 0,
+        }
+    }
+}
+
+#[allow(dead_code)]
+impl Account {
+    pub fn new(balance: u64, sequence: u64) -> Account {
+        Account {
+            balance,
+            sequence,
+        }
+    }
+
+    // Temporary measures for build.
+    // This function will be removed when the structure of the context and conversion logic is completed.
+    pub fn to_vec(&self) -> Vec<u8> {
+        vec![0]
+    }
+}
+
+type Bytes = Vec<u8>;
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize)]
+pub struct Transaction {
+    pub seq: u64,
+    pub fee: u64,
+    pub network_id: NetworkId,
+    pub action: Action,
+}
+
+impl Transaction {
+    pub fn hash(&self) -> H256 {
```

Comment:
Is there an actual usage for this method in the module?

Reply:
As I know, It's used for verifying signature when transaction is executed.

Reply:
Ah, Ok.